docs/appendix.rst

```diff
@@ -137,6 +137,20 @@ This documentation uses a few special terms to refer to Python types:
       No tilde expansion is done, but ``*``, ``?``, and character ranges
       expressed with ``[]`` will be correctly matched.
 
+   traditional operations
+      The CIM-XML operations originally defined by the DMTF in :term:`CIM-XML`
+      operations defined in the DMTF specification CIM Operations Over HTTP
+      (:term:`DSP0200`) for requesting multiple instances including
+      ``EnumerateInstances```, ``EnumerateInstanceNames``, ``Referencess``,
+      ``ReferenceNames``, ``Associators``, AssociatorNames``, and `ExecQuery`.
+      These are monolitic operations and expect the WBEM server to prepare
+      complete reponses before returning any results because they are
+      monolithic and also because the response must be either the response
```

Comment:
"monolithic" used twice. Could be streamlined.

Reply:
misspelled also.

Rewrote paragraph